src/modules/utilities/Information.ts

```diff
@@ -1,4 +1,6 @@
-class Information {
+import introJs from '../../../node_modules/intro.js/intro';
```

Comment:
Can be shortened to "import introJs from 'intro.js';"
Node_modules is automatically compiled by NPM so it's best to avoid referencing it directly

Reply:
I already tried it exactly as you said and it messes up the intro.

Reply:
Making this change locally doesn't seem to break anything for me? What is broken when you do this?

Reply:
You know, i tried it again just now and it worked fine. Idk what was going on last night/this morning.